hollow-jsonadapter/build.gradle

```diff
@@ -1,12 +1,14 @@
 apply plugin: 'java'
- 
+
 dependencies {
     compile project(':hollow')
 
-    compile 'com.fasterxml.jackson.core:jackson-core:2.4.3'
-    compile 'com.fasterxml.jackson.core:jackson-databind:2.4.3'
+    compile 'com.fasterxml.jackson.core:jackson-core:2.6.0'
+    compile 'com.fasterxml.jackson.core:jackson-databind:2.6.0'
     compile 'commons-io:commons-io:2.4'
     compile 'commons-lang:commons-lang:latest.release'
+    compile 'org.ow2.asm:asm:6.2.1'
```

Comment:
I have some concern about depending on asm. It can be problematic when there conflicts with other libraries depending on other versions. This is required for accessing methods in declaration order, but it is not clear to me it achieves the desired behaviour (if declaration order is source declaration order).